bin/compile

```diff
@@ -93,22 +107,4 @@ echo " done"
 mkdir -p "$build/.profile.d"
 cat << EOF > "$build/.profile.d/boot-env.sh"
 export PATH="\$HOME/.bootbin:\$PATH"
-
-limit=\$(ulimit -u)
-case \$limit in
-256)   # 1X Dyno
-  default_java_opts="-Xmx384m -Xss512k"
-  ;;
-512)   # 2X Dyno
-  default_java_opts="-Xmx768m"
-  ;;
-32768) # PX Dyno
-  default_java_opts="-Xmx4g"
-  ;;
-*)
-  default_java_opts="-Xmx384m -Xss512k"
-  ;;
-esac
-
-export BOOT_JVM_OPTIONS=\${BOOT_JVM_OPTIONS:-"\${default_java_opts} -Dfile.encoding=UTF-8"}
```

Comment:
I'm not very well versed in JDK magic env vars. Does boot executable pay attention to JAVA_TOOL_OPTIONS? @jkutner